user-storage-simple/src/main/java/org/keycloak/quickstart/writeable/PropertyFileUserStorageProvider.java

```diff
@@ -265,7 +265,7 @@ public boolean updateCredential(RealmModel realm, UserModel user, CredentialInpu
         if (!input.getType().equals(CredentialModel.PASSWORD)) return false;
         UserCredentialModel cred = (UserCredentialModel)input;
         synchronized (properties) {
-            properties.setProperty(user.getUsername(), cred.getValue());
+            properties.setProperty(user.getUsername(), cred.getChallengeResponse());
```

Comment:
Remove this - we need to fix API issues on the server side

Reply:
I sent PR to fix this on server side. If you have a chance to review, it is here keycloak/keycloak#6795

Reply:
Resolving this comment, see https://github.com/keycloak/keycloak-quickstarts/pull/166#discussion_r380572963